brave/browser-laptop
# Polish compact Bravery panel

Closes #9077 
Addresses #9016 

Test Plan:
1. Visit twitter.com
2. Make sure only counts of blocked resources are clickable

Submitter Checklist:


Reviewer Checklist:

Tests
